openedx/features/edly/utils.py

```diff
@@ -0,0 +1,18 @@
+from openedx.features.edly.models import EdlyUserProfile
+
+
+def user_has_edly_organization_access(request):
+    """
+    Return True if the edly-sub-organization related to request URL is listed in get_linked_edly_sub_organizations.
+    """
```

Comment:
```
"""
    Return True if the edly-sub-organization of the requested URL belongs to the user's linked edly-sub-organizations.
"""
```
Sounds more readable to me. We can remove the edly word as well if make sense to you.

Reply:
Done